tests/mlp_models.py

```diff
@@ -27,10 +27,10 @@ def dynamic_classifier(
         hidden = Dense(layer_size, activation="relu")(hidden)
 
     if target_type_ == "binary":
-        compile_kwargs["loss"] = compile_kwargs["loss"] or "binary_crossentropy"
+        compile_kwargs["loss"] = "binary_crossentropy"
```

Comment:
Unfortunately it's not that simple... The original intention was "use the inferred loss unless it is explicitly overriden", and there are indeed tests that explicitly override it to test various things (wrong loss, no loss, etc.)